src/main/java/de/tum/cit/ase/ares/api/aop/java/instrumentation/pointcut/JavaInstrumentationBindingDefinitions.java

```diff
@@ -0,0 +1,99 @@
+package de.tum.cit.ase.ares.api.aop.java.instrumentation.pointcut;
+
+import de.tum.cit.ase.ares.api.aop.java.JavaSecurityTestCaseSettings;
+import de.tum.cit.ase.ares.api.aop.java.instrumentation.advice.JavaInstrumentationAdviceToolbox;
+import de.tum.cit.ase.ares.api.aop.java.instrumentation.advice.JavaInstrumentationDeletePathAdvice;
+import de.tum.cit.ase.ares.api.aop.java.instrumentation.advice.JavaInstrumentationExecutePathAdvice;
+import de.tum.cit.ase.ares.api.aop.java.instrumentation.advice.JavaInstrumentationReadPathAdvice;
+import de.tum.cit.ase.ares.api.aop.java.instrumentation.advice.JavaInstrumentationOverwritePathAdvice;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.dynamic.ClassFileLocator;
+import net.bytebuddy.dynamic.DynamicType;
+import net.bytebuddy.dynamic.loading.ClassInjector;
+import net.bytebuddy.utility.JavaModule;
+
+import java.security.ProtectionDomain;
+import java.util.List;
+import java.util.Map;
+
+
+public class JavaInstrumentationBindingDefinitions {
+
+    private JavaInstrumentationBindingDefinitions() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    //<editor-fold desc="Tools">
+    private static DynamicType.Builder<?> createBinding(
```

Comment:
No javadoc in the JavaInstrumentationBindingDefinitions hard to understand from a not knowing persons perspective